haskell/nix.bzl

```diff
@@ -0,0 +1,166 @@
+"""Loading of haskell packages from nixpkgs"""
+
+load(
+    "@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
+    "nixpkgs_package",
+)
+
+def haskell_nixpkgs_package(
+        name,
+        attribute_path,
+        nix_file_deps = [],
+        repositories = {},
+        build_file_content = None,
+        build_file = None,
+        *args,
+        **kwargs):
+    """Load a single haskell package.
+    The package is expected to be in the form of the packages generated by
+    `genBazelBuild.nix`
+    """
+    repositories = {
+        "bazel_haskell_wrapper": "@io_tweag_rules_haskell//haskell:nix/default.nix",
+    } + repositories
+
+    nixpkgs_args = dict(
+        name = name,
+        attribute_path = attribute_path,
+        build_file_content = build_file_content,
+        nix_file_deps = nix_file_deps + ["@io_tweag_rules_haskell//haskell:nix/default.nix"],
+        repositories = repositories,
```

Comment:
In playing with this I've realised (I believe) that it depends on the `allow-multiple-repositories` branch of `rules_nixpkgs` being merged first; is this right?

Reply:
Oh you're right, I've forgotten to write it explicitely. I'll add it to the description of the PR